queries/python/highlights.scm

```diff
@@ -176,7 +176,7 @@
 (escape_sequence) @string.escape
 
 ; doc-strings
-(expression_statement (string) @spell)
+(expression_statement (string) @string.documentation @spell)
```

Comment:
Does this capture all the docstrings?

Reply:
![image](https://user-images.githubusercontent.com/29718261/222329154-44436a8f-da07-45a1-9002-c5a5302a21c2.png)

Yes, but maybe it captures too much? Like it'll capture random floating strings..although tbf they don't do anything code-wise so I guess that's fine to assume it's a docstring

Reply:
Shouldn't it only capture the `"""` right after a class/function definition?
Example of the tree with docstring
![image](https://user-images.githubusercontent.com/29790821/222340430-509c2afc-7f25-4d2b-9275-128b3014f599.png)
Something like
```scheme
(function_definition
  body: 
    (block
      . (expression_statement) @string.doc))
```
 would be better imo

Reply:
Yes, and similar for classes and whatnot. Basically flip the Julia queries ;)